tools/src/commands/GenerateStoriesApp.ts

```diff
@@ -0,0 +1,87 @@
+import { Command } from '@expo/commander';
+import spawnAsync from '@expo/spawn-async';
+import fs from 'fs-extra';
+import path from 'path';
+
+import { EXPO_DIR } from '../Constants';
+import {
+  GenerateBareAppOptions,
+  action as generateBareApp,
+  getDirectories,
+  getPackagesToSymlink,
+  symlinkPackages,
+} from './GenerateBareApp';
+
+async function action(packageNames: string[], options: GenerateBareAppOptions) {
+  const { projectDir, workspaceDir } = getDirectories(options);
+  await generateBareApp(packageNames, options);
+  await addNativeDependencies({ projectDir });
+  await createMetroConfig({ projectRoot: projectDir, ...options });
+  await rewriteAppFile({ projectDir, appName: options.name });
+
+  const packagesToSymlink = await getPackagesToSymlink({ packageNames, workspaceDir });
+  await symlinkPackages({ packagesToSymlink, projectDir });
+}
+
+async function addNativeDependencies({ projectDir }: { projectDir: string }) {
+  const pkgPath = path.resolve(projectDir, 'package.json');
+  const pkg = await fs.readJSON(pkgPath);
+
+  pkg.dependencies['react-native-safe-area-context'] = '4.2.4';
+  pkg.dependencies['react-native-screens'] = '~3.11.1';
+  pkg.dependencies['react-native-svg'] = '12.3.0';
+  pkg.dependencies['react-native-gesture-handler'] = '2.4.2';
+  pkg.dependencies['expo-stories'] = '1.0.1';
```

Comment:
Hmm, this looks like another place we have to keep in mind when upgrading dependencies. Maybe we could read these versions from some single source of truth, like `bundledNativeModules.json`?

Reply:
Yeah, we definitely don't want to maintain this, so any other way of getting these versions would be great

Reply:
thank you for the review!